spec/api/people/show_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+require "spec_helper"
+
+RSpec.describe "people#show", type: :request do
```

Comment:
If you move to file to `spec/requests/api/...` then you also don't need to specify the `type:`. Also I'd use the conroller class in `describe`

```suggestion
RSpec.describe FoobarController do
```
On the other hand the existing api specs are done exactly like this... So lets keep it.